YAML/YAML.sublime-syntax

```diff
@@ -527,16 +527,23 @@ contexts:
             - match: '{{_flow_scalar_end_plain_out}}'
               pop: true
     - match: :(?=\s|$)
-      scope: punctuation.separator.mapping.key-value.yaml
+      scope: punctuation.separator.key-value.mapping.yaml
 
   comment:
     # http://www.yaml.org/spec/1.2/spec.html#comment//
     - match: | # l-comment
         (?x)
-        (?: ^ [ \t]* | [ \t]+ )
-        (?:(\#) \p{Print}* )?
-        (\n|\z)
-      scope: comment.line.number-sign.yaml
+        (?: ( ^ [ \t]* ) | [ \t]+ )
+        (?=#\p{Print}*$)
```

Comment:
Ah correct, in the TextMate format grammar I had this as a non-extended and only changed it because the original rule here was extended.